phpBB/includes/functions.php

```diff
@@ -5448,22 +5436,6 @@ function phpbb_user_session_handler()
 }
 
 /**
-* Check if PCRE has UTF-8 support
-* PHP may not be linked with the bundled PCRE lib and instead with an older version
-*
-* @return bool	Returns true if PCRE (the regular expressions library) supports UTF-8 encoding
-*/
-function phpbb_pcre_utf8_support()
```

Comment:
I have no opinion on that matter. There are no calls to `phpbb_pcre_utf8_support()` left in the source but if you think an extension may call that function it can be moved to `functions_compatibility.php` and optionally replaced with a stub.